### PR TITLE
docs: update readme contributing

### PR DESCRIPTION
**Pull request by wangqianwen0418**

### CODE REVIEW

**Comment by wangqianwen0418** [comment]
Related to https://github.com/gosling-lang/gosling.js/pull/503#pullrequestreview-777422173

@sehilyi, I have no idea why the sematic PR cannot pass. any idea?

**Comment by sehilyi** [comment]
> sehilyi, I have no idea why the sematic PR cannot pass. any idea?

Apologize, this is not well documented. The title of PRs uses [commitlint](https://github.com/conventional-changelog/commitlint#what-is-commitlint) conventions, so you can see errors if the title is not following the convention.

File: README.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ We welcome and greatly appreciate your contribution to this project! Please read
   title={Gosling: A Grammar-based Toolkit for Scalable and Interactive Genomics Data Visualization},
   author={Sehi L'Yi and Qianwen Wang and Fritz Lekschas and Nils Gehlenborg},
   year={2021},
-  publisher={OSF Preprints},
```
Comment (sehilyi) [inline README.md:52]: Thank you for updating this!

**Review by sehilyi — APPROVED** [review]
Nice! 

I just ran `yarn format` and made a minor update.

**Pull request merged**